### ISSUE TITLE
"lines" in Ratchet & Clank 1, 2

### ISSUE THREAD

**Issue by EvilPlayer1** [PCSX2/pcsx2#11466]
### Describe the Bug

Along the edges on the left and at the top of the screen can be seen strange lines, which broke image rendering. The best way to see them is to put any texture in the top left corner of the screen.
![Screenshot 25-06-2024 19-03-21](https://github.com/PCSX2/pcsx2/assets/90133207/d2526a73-e7d7-4f09-a23c-8c7a2b1f56df)
![Screenshot 25-06-2024 19-04-05](https://github.com/PCSX2/pcsx2/assets/90133207/faa3e999-2154-439c-88d1-f8d1fe41dc32)

### Reproduction Steps

Put any texture in the top left corner of the screen and see this "ghosting" effect.

### PCSX2 Revision

v1.7.5935

### Operating System

Windows 11

### CPU

i7-11700

### GPU

GTX 1660 SUPER

### GS Settings

Almost default, no graphic hacks

### Emulation Settings

All recommended settings

**Comment by Mrlinkwii** [comment]
can i have a gs dump

**Comment by EvilPlayer1** [comment]
[Ratchet & Clank 2_ Going Commando_SCUS-97268_20240626004312.gs.zip](https://github.com/user-attachments/files/15979657/Ratchet.Clank.2_.Going.Commando_SCUS-97268_20240626004312.gs.zip)

**Comment by bigol83** [comment]
This is how it looks for me with default settings, only 3x resolution

![Ratchet   Clank 2 - Going Commando_SCUS-97268_20240625234936](https://github.com/PCSX2/pcsx2/assets/38129260/00c961bd-aaa0-4632-8556-684483fb7121)

No issue on the top left.

**Comment by EvilPlayer1** [comment]
> This is how it looks for me with default settings, only 3x resolution
> 
> ![Ratchet Clank 2 - Going Commando_SCUS-97268_20240625234936](https://private-user-images.githubusercontent.com/38129260/342944525-00c961bd-aaa0-4632-8556-684483fb7121.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTkzNTI2NjEsIm5iZiI6MTcxOTM1MjM2MSwicGF0aCI6Ii8zODEyOTI2MC8zNDI5NDQ1MjUtMDBjOTYxYmQtYWFhMC00NjMyLTg1NTYtNjg0NDgzZmI3MTIxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjI1VDIxNTI0MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNmNTZkMjI4MDUwYTJiMWUwNWNiNjI2OTZmMzU4NzZmZDY5MTNkNmU5NmZlM2IyZmMwMWRlNjg1ZjlhMjcxYzYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.EJBEGG_NxZ9vakjj027S91fLLHgf6pcX6CP6VEEXNs8)
> 
> No issue on the top left.

![342944525-00c961bd-aaa0-4632-8556-684483fb7121](https://github.com/PCSX2/pcsx2/assets/90133207/30885782-bd49-4023-8549-9f0964c82047)
flashlight texture is damaged

**Comment by bigol83** [comment]
Software renderer shows that too 

![Ratchet   Clank 2 - Going Commando_SCUS-97268_20240625235401](https://github.com/PCSX2/pcsx2/assets/38129260/70484d47-66e2-4fd4-913c-063a4494dc6f)

**Comment by refractionpcsx2** [comment]
oh I think that's the clamp, yeah nothing you can do about that right now, sorry, you can crop it out

**Comment by EvilPlayer1** [comment]
> oh I think that's the clamp, yeah nothing you can do about that right now, sorry, you can crop it out

yeah, it helped. 2 - left, 1 - top. The same bug was in R&C 3 and 4 earlier, when pause menu wasn't stretched to fullscreen correctly and you could see default image through those lines. For now it seems to be fixed.
